specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2016-10-01/keyvault.json

```diff
@@ -16,6 +16,7 @@
           "Vaults"
         ],
         "operationId": "Vaults_CreateOrUpdate",
+        "x-ms-long-running-operation": true,
```

Comment:
Can you verify if `Vaults_Update` is also a long running operation

Reply:
Vault_Update isn't expected to be long-running. The issue I'm addressing here is that vault creation happens in <1 second, but it may take 15+ seconds for DNS to successfully resolve when a vault is first created. On the service-side, I've implemented a 15 second delay for returning a terminal provisioningState (so the initial PUT and all GETs in the first 15 seconds return a non-terminal and currently unspecified provisioningState). The goal is to ensure that a PowerShell script that calls New-AzureRmKeyVault followed by Set-AzureKeyVaultSecret won't run into errors due to DNS propagation delays. The New-AzureRmKeyVault cmdlet will end up handling the long-running operation, so even if the script does other vault updates via Set-AzureRmKeyVaultAccessPolicy, the desired delay will have already occurred.

Reply:
Thanks for clarifying.